pkg/osquery/runtime/runtime_helpers_windows.go

```diff
@@ -80,3 +82,39 @@ func isExitOk(err error) bool {
 	}
 	return false
 }
+
+func getProcessesHoldingFile(ctx context.Context, pathToFile string) ([]*process.Process, error) {
```

Comment:
I think this is a fairly expensive call. Might be okay, given how infrequent it occurs. But we should know that it's not cheap

Reply:
It takes about 6 seconds in test, which is definitely not nothing. But we'll only hit this if the lock file exists -- and in that case, we'll never successfully start up osquery anyway, so the delay is a moot point. I figure we can remove this once we have more info from the logs to figure out how to detect/remediate.

Reply:
Yeah...